stable/firehose/templates/deployment.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "firehose.fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels: {{- range $key, $value := .Values.labels }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ include "firehose.fullname" . }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap-firehose.yaml") . | sha256sum }}
+      labels:
+        app: {{ include "firehose.fullname" . }}
+        {{- range $key, $value := .Values.labels }}
+        {{ $key }}: {{ $value }}
+        {{- end }}
+    spec:
+      containers:
+      - args:
+        {{- range .Values.firehose.args }}
+        - {{ . }}
+        {{- end }}
+        name: {{ .Chart.Name }}
+        image: "{{ .Values.firehose.image.repository }}:{{ .Values.firehose.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.firehose.image.pullPolicy }}
+        volumeMounts:
+        {{- if index .Values "init-firehose" "enabled" }}
+        - mountPath: /work-dir
+          name: workdir
+        {{- end }}
+        resources:
+          limits:
+            cpu: {{ .Values.firehose.resources.limits.cpu }}
+            memory: {{ .Values.firehose.resources.limits.memory }}
+          requests:
+            cpu: {{ .Values.firehose.resources.requests.cpu }}
+            memory: {{ .Values.firehose.resources.requests.memory }}
```

Comment:
We can get this hierarchy directly [like this](https://github.com/odpf/charts/blob/main/stable/siren/templates/deployment.yaml#L57) instead of specifying granularly